lib/rspec/expectations/fail_with.rb

```diff
@@ -26,6 +26,10 @@ def fail_with(message, expected=nil, actual=nil)
 
         RSpec::Support.notify_failure(RSpec::Expectations::ExpectationNotMetError.new message)
       end
+
+      def call(object)
+        RSpec::Matchers::Composable.surface_descriptions_in(object)
+      end
```

Comment:
This exposes `RSpec::Expectations.call` as a method, which is undesirable.  It's not clear what "calling `RSpec::Expectations`" should mean.  Instead, can you define a module like this and pass that instead of `self` above?

``` ruby
module RSpec
  module Expectations
    module DifferObjectPreparer
      def self.call(object)
        RSpec::Matchers::Composable.surface_descriptions_in(object)
      end
    end
  end
end
```